Stats.xcodeproj/xcshareddata/xcschemes/WidgetsExtension.xcscheme

```diff
@@ -102,7 +102,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
-      askForAppToLaunch = "Yes"
```

Comment:
what the reason you remove this line and how this related to the Finnish translation?